articles/storage/storage-use-azcopy.md

```diff
@@ -250,6 +250,15 @@ After the copy operation, the target container will include the blob and its sna
     abc (2013-02-25 080757).txt
     abc (2014-02-21 150331).txt
 
+### Copy all blobs across Storage accounts
+
+```azcopy
+    AzCopy /Source:https://sourceaccount.blob.core.windows.net/mycontainer1 /Dest:https://destaccount.blob.core.windows.net/mycontainer2 /SourceKey:key1 /DestKey:key2 /S
+```
+
+If option `/S` is specified, then AzCopy will copy all blobs in source container to destination container including its contained containers recursively. You can specify `/Pattern` option together in order to copy matched file pattern.
```

Comment:
You can't nest containers, so there are no "contained containers". This is explained in Upload all files, so if you want to, maybe you could say this here: 
Specifying option `/S` uploads the contents of the specified container recursively. See [upload all files](#upload-all-files) for more information and an example.
This will link to the other section.